File: nelio-alves/algoritmo-e-logica-de-programacao/python/exercicios/cada_linha.py
N: int; maior: int;

N = int(input("Qual a ordem da matriz? "));

mat: [[int]] = [[0 for x in range(N)] for x in range(N)]

for i in range(N):
	for j in range(N):
		mat[i][j] = int(input(f"Elemento [{i},{j}]: "))

print("\nMATRIZ DIGITADA:")
for i in range(N):
    for j in range(N):
        print(f"{mat[i][j]} ", end="")
    print()

print("\nMAIOR ELEMENTO DE CADA LINHA:")
for i in range(N):
	maior = mat[i][0]
	for j in range(1, N):
		if maior < mat[i][j]:
			maior = mat[i][j]

	print(maior)

File: nelio-alves/algoritmo-e-logica-de-programacao/c/exercicios/cada_linha.c
#include<stdio.h>

int main() {
    int N, maior;

    printf("Qual a ordem da matriz? ");
    scanf("%d", &N);

    int mat[N][N];

    for (int i = 0; i < N; i++) {
        for (int j = 0; j < N; j++) {
            printf("Elemento [%d,%d]: ", i, j);
            scanf("%d", &mat[i][j]);
        }
    }

    printf("\nMATRIZ DIGITADA:\n");
    for (int i = 0; i < N; i++) {
        for (int j = 0; j < N; j++) {
            printf("%d ", mat[i][j]);
        }
        printf("\n");
    }

    printf("\nMAIOR ELEMENTO DE CADA LINHA:\n");
    for (int i = 0; i < N; i++) {
        maior = mat[i][0];
        for (int j = 0; j < N; j++) {
            if (maior < mat[i][j]) {
                maior = mat[i][j];
            }
        }
        printf("%d\n", maior);
    }

    return 0;
}
